hacks/common.js

```diff
@@ -104,7 +104,7 @@ DBQuery.prototype.shellPrint = function(){
         if (typeof _verboseShell !== 'undefined' && _verboseShell) {
             var time = new Date().getTime() - start;
             var slowms = getSlowms();
-            var fetched = "Fetched " + n + " record(s) in ";
+            var fetched = "Fetched " + n + " record" + (n !== 1 ? "s" : "") + " in ";
```

Comment:
This might be better as `(n === 1 ? "" : "s")`, what do you think?

First we get the triple equality check (making sure its a 1 number -- not string), and we convert the expression to use a positive condition which might make it easier to grok.